packages/api/src/webhooks/cannon.ts

```diff
@@ -69,7 +77,7 @@ export default class WebhookCannon {
     event.lastInterval = this.calcBackoff(event.lastInterval);
     event.status = "pending";
     event.retries = event.retries ? event.retries + 1 : 1;
-    this.db.queue.updateMsg(event);
+    this.queue.delayedEmit(event, event.lastInterval);
   }
 
   async _fireHook(
```

Comment:
(closest place I could comment where I actually meant this, on [line 135](https://github.com/livepeer/livepeer-com/pull/453/files#diff-8942d451dd2aa33dc5fdc9410a004c7011e39201e287b71a009216c88830ebf7L135))

There, we are doing a sleep before firing a webhook, but I understood that was a necessary logic only for the Postgres queue but not for RabbitMQ, since we handle retries in it with the message TTL+deadletter queue instead. 

So, do we still need that sleep on can we just deliver immediately, since the event will have been delayed already? I wonder what kind of tests could have caught this as well 🤔